### DISCUSSION TITLE
Cannot Build

### DISCUSSION THREAD

**Discussion by smchargue** [pnp/sp-dev-fx-webparts#2450] [Q&A]
Cannot build - all day effort. 
The instructions 
```
npm install
``` 
are completely insufficient. 

![image](https://user-images.githubusercontent.com/4064621/160149522-2d8b2bb4-185b-47f1-9932-bb54078a22cd.png)

**Comment by gretchunkim** [comment]
You'll need to resolve all npm err messages, and try to look for project specific installation requirement to see which one need update/match  version. 

At general, it's confusince since it's not mentioned anywhere directly on the documentation when I started but you'll need to install npm install twice. one for global and one for project specific.

**Reply by melliflua** [reply]
yeah same issue for me smchargue; npm install -g followed by npm i does not resolve, as you suggest gretchunkim.  I will try downgrade from node 16 to 14 (well ... rather nvm switch to 14); I am such a newb.

**Reply by melliflua** [reply]
Yep, switch with nvm to node version 14 allowed npm install (or downgrade to 14 if you can't/don't have nvm)
